mlflow/openai/__init__.py

```diff
@@ -410,6 +417,12 @@ def save_model(
 
     if is_in_databricks_runtime():
         if scope := MLFLOW_OPENAI_SECRET_SCOPE.get():
+            url = "https://docs.databricks.com/en/machine-learning/model-serving/store-env-variable-model-serving.html"
+            warnings.warn(
+                "Specifying secrets for model serving with `MLFLOW_OPENAI_SECRET_SCOPE` is "
```

Comment:
Shall we call out it's deprecated in "Databricks runtime"?

Reply:
@chenmoneygithub I would add `Databricks runtime` if `MLFLOW_OPENAI_SECRET_SCOPE` was used somewhere else.

Reply:
> Specifying secrets for model serving with `MLFLOW_OPENAI_SECRET_SCOPE` is deprecated in Databricks runtime.

Reply:
added

Reply:
and removed it. `deprecated in Databricks runtime` sounds a bit strange to me.